src/lib.rs

```diff
@@ -261,7 +261,7 @@ pub trait Colorize {
     fn custom_color<T>(self, color: T) -> ColoredString
     where
         Self: Sized,
-        T: Into<CustomColor>,
+        T: Into<RGB<u8>>,
```

Comment:
Looking through the `rgb` crate, the `RGB8` type alias exists to simplify this a bit.

Reply:
I would say that Rgb<u8> is probably fine, i did switch it to using the more idiomatic type for the struct though.